### ISSUE TITLE
Getting Error when trying to import

### ISSUE THREAD

**Issue by MilanDeruelle** [mattlubner/Import-to-iTunes#1]
Title says it all, selecting folder, selecting. "Finder -> Services -> Import to iTunes" and it gives me a generic error message

**Comment by mattlubner** [comment]
Yep, dang, I can reproduce it too. Getting an `The action “Run Shell Script” encountered an error.` error message.

More troublesome is that Automator crashes when I try to open the workflow. And even more disconcerting, if I create a new workflow in Automator and then try to drag the "Import Files into iTunes" action into the new workflow, Automator promptly crashes.

This has the smell of an iTunes bug to me.

**Comment by MilanDeruelle** [comment]
I can open it in Automator though, just getting the Shell Script error... not sure if you're right about the iTunes bug

**Comment by mattlubner** [comment]
Huh! Can you post what version of macOS and iTunes you're on?

I wonder why my Automator is crashing… 😕

**Comment by cassioac** [comment]
The error I get is:

The action “Run Shell Script” encountered an error: “2017-10-22 10:03:17.540 defaults[4110:420047] 
The domain/default pair of (com.apple.iApps, iTunesRecentDatabases) does not exist”


High Sierra, iTunes 12.7.0.166

**Comment by mattlubner** [comment]
@cassioac My Automator.app seems to be corrupted (it still crashes on launch) so I won't be able to address this crash until I can do a clean install of macOS… if you're able to isolate the issue, please feel free to submit a pull request though!
